api.go

```diff
@@ -130,6 +131,74 @@ func (server *ApiServer) ProxyCreate(response http.ResponseWriter, request *http
 	}
 }
 
+func (server *ApiServer) Populate(response http.ResponseWriter, request *http.Request) {
+	input := []struct {
+		Proxy
+		Enabled *bool `json:"enabled"` // Overrides Proxy field to make field nullable
+	}{}
```

Comment:
The type is only ever used in this one function and array, so I didn't think it warranted naming. It's similar to other json marshalling code we have that uses one-time objects.